Assets/MRTK/Core/Utilities/Gltf/Schema/GltfAccessor.cs

```diff
@@ -100,5 +105,22 @@ public class GltfAccessor : GltfChildOfRootProperty
         /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/bufferView.schema.json
         /// </summary>
         public GltfBufferView BufferView { get; internal set; }
+
+        public void OnAfterDeserialize()
+        {
+            if (Enum.TryParse(componentType, out GltfComponentType result))
+            {
+                ComponentType = result;
+            }
+            else
+            {
+                ComponentType = default;
+            }
+        }
+
+        public void OnBeforeSerialize()
+        {
+
```

Comment:
Should these be implemented?

Reply:
I do not think so as we are not doing serialization with GLTF objects at the moment.

Reply:
Oh, true!